### ISSUE TITLE
ambiguousity of cout/cin 

### ISSUE THREAD

**Issue by lavkesh1709** [microsoft/vscode-cpptools#11124]
Type: <b>Bug</b>

"cout" is ambiguousC/C++(266)
std::ostream std::cout
/ Linked to standard output"
This has been dispalyed on the screen about the ambiguoity of cin/cout.

Extension version: 1.16.2
VS Code version: Code 1.79.2 (695af097c7bd098fbf017ce3ac85e09bbc5dda06, 2023-06-14T08:57:04.379Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 5600H with Radeon Graphics          (12 x 3294)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.35GB (1.63GB free)|
|Process Argv|--crash-reporter-id 52b46b13-1f2f-471c-85c3-4eb4c9ea9095|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263:30773603
vscaac:30438847
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyonecf:30548226
vscccc:30610679
2e4cg342:30602488
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30771522
pythongtdpath:30769146
e440d664:30770001
pythonidxptcf:30772540
pythondjangotscf:30772537
pythonnocebcf:30773527
e537b577:30772215

```

</details>

**Comment by sean-mcmanus** [comment]
We're looking into it, see https://github.com/microsoft/vscode-cpptools/issues/11122 .

**Comment by michelleangela** [comment]
This is now fixed in [release version 1.16.3](https://github.com/microsoft/vscode-cpptools/releases/tag/v1.16.3).
